.github/workflows/publish.yml

```diff
@@ -54,7 +54,7 @@ defaults:
 jobs:
     unit-tests:
         if: |
-            !contains(fromJson(vars.SKIP_UNIT_TESTS), inputs.package) &&
```

Comment:
You could make `SKIP_UNIT_TESTS` an env variable in the workflow like:
```
env:
   SKIP_UNIT_TESTS='["dbt-tests-adapter"]

Reply:
It crossed my mind to do that but I was concerned that it would be over-engineering it again, which is sort of what led here. But since you're agreeing with that, I'll do that.